_posts/2021-6-30-introducing-community-projects.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: post
+title: Introducing the community projects page
+authors: 
+  - elifish
+date: 2021-06-30
+categories:
+  - community
+twittercard:
+  description: "Many open source projects benefit from the ecosystem of software created on top of or for the project. As a first steps towards having a robust ecosystem, opensearch.org now has added a community projects page to highlight projects created for the OpenSearch community by the OpenSearch community... "
+---
+
+Many open source projects benefit from the ecosystem of software created on top of or for the project. As an example, the Node Package Manager (NPM) has been instrumental in helping JavaScript be more approachable. It hosts a variety of modules that solve problems people face during the development of JavaScript software. Users can quickly search for these modules install them and incorporate them directly into their JavaScript project. Having a similar ecosystem for OpenSearch will help all of us get more out of our own OpenSearch deployments. As a first step, I wanted to share that [opensearch.org](/) now has a [community projects page](/community_projects/). The goal of the community projects page is to highlight projects created for OpenSearch.
```

Comment:
`As an example, the Node Package Manager (NPM) has been instrumental in helping JavaScript be more approachable.` <- Passive voice

`The goal of the community projects page is to highlight projects created for OpenSearch.` <- this is a bit awkward

Reply:
reworded. Let me know if the new wording works.